README.md

```diff
@@ -704,6 +704,99 @@ significantly compared to firing the same commands in a loop without waiting for
 the result! See the benchmarks for further comparison. Please remember that all
 commands are kept in memory until they are fired.
 
+## Optimistic Locks
+
+Using `multi` you can make sure your modifications run as a transaction, but you 
+can't be sure you got there first. What if another client modified a key while 
+you were working with it's data?
+
+To solve this, Redis supports the [WATCH](https://redis.io/topics/transactions) 
+command, which is meant to be used with MULTI:
+
+```js
+var redis  = require("redis"),
+    client = redis.createClient();
```

Comment:
Please add something to the arguments of `redis.createClient`, even if it's just an empty object indicating that something _should_ be there. No arguments means that you have no password on your Redis instance and it's a huge security issue, so we always try to illustrate best practices.